uber/site_sections/registration.py

```diff
@@ -890,6 +890,26 @@ def watch_list(self, session, message='', **params):
             'message': message
         }
 
```

Comment:
I'm somewhat ambivalent about the number of pages that people create to do the equivalent of running a one-time script.  On the one hand, it's sometimes an expedient way to get something pushed out, especially when the person who wants to run the thing can't just SSH into the server.  And now that we're starting to split this kind of thing out into per-event plugins, it's not like it'll even clutter up the core repo with one-off code, sooo.... I guess it's okay.

On the other hand, I worry that in some cases it's trying to solve an organizational problem with a technical one.  I'm probably just overreacting though :)